modules/040-node-manager/hooks/handle_draining.go

```diff
@@ -107,9 +123,19 @@ func handleDraining(input *go_hook.HookInput, dc dependency.Container) error {
 	snap := input.Snapshots["nodes_for_draining"]
 	for _, s := range snap {
 		dNode := s.(drainingNode)
```

Comment:
It would be a little more obvious if at the loop beginning we had additional `if` to skip irrelevant nodes. (not draining and not drained)

Reply:
We had this if one line below

Reply:
I see the `continue` line (L111). It is just not that obvious.